ddtrace/propagation/http.py

```diff
@@ -154,7 +158,16 @@ def _inject(span_context, headers):
             log.debug("tried to inject invalid context %r", span_context)
             return
 
-        headers[HTTP_HEADER_TRACE_ID] = str(span_context.trace_id)
+        if span_context.trace_id > _MAX_UINT_64BITS:
+            # set lower order 64 bits in `x-datadog-trace-id` header. For backwards compatibility these
+            # bits should be converted to a base 10 integer.
+            headers[HTTP_HEADER_TRACE_ID] = str(get_64_lowest_order_bits_as_int(span_context.trace_id))
+            # set higher order 64 bits in `_dd.p.tid` to propagate the full 128 bit trace id.
```

Comment:
We agreed upon this across tracers? Just want to make sure for propagation

Reply:
Yup, this is defined in the RFC and already implemented in Ruby